### DISCUSSION TITLE
New QField file access harder to handle?

### DISCUSSION THREAD

**Discussion by OBVYamaska** [opengisch/QField#2720] [Q&A]
Am I the only one the can't really grasp what is going on with the recently updated File access in QField?
I find it way harder than before to simply locate my files and start projects... a concept that should be pretty straight forward!
Mainly, it seems that I no longer have access to the "root" of my device, only to specific folders that are deep in my phone. So the previous "drag and drop" process no longer applies.

Also, when I add a new folder connection to access the added projects, it remains static. So if I add other projects to the repertory with cable connection, they won't show in the app; I have to remove the previous folder connection and make a new one.

This makes everything way harder to work with... especially for less techy coworkers!

Oh, and there is a slight glitch with the delete window: when I try to remove a folder connection, the two options (which are probably Yes/No) are invisible. Clicking where they should be works though.
![Screenshot_20220406-122542](https://user-images.githubusercontent.com/101751953/162292739-0da25b18-d1b1-461d-8516-896e6cc08cd8.jpg)

**Comment by nirvn** [comment]
@OBVYamaska , if you haven't done so, please read this blog post on storage access using Android >= 10 (https://www.opengis.ch/2022/03/05/qfield-users-sit-down-we-need-to-talk-about-storage-access-on-android%ef%bf%bc/).

_TLDR: Since November 2021, Google has enforced new storage access limitations for apps published on its Play store which prohibits direct storage access on Android 11 and above forcing QField to adapt and rely on importing projects and datasets to access those._

If you have time, read the whole thing, it's informative :) we are aiming at further improving the new storage access paradigm in QField 2.1. Consider your [ YES / NO ] glitch fixed already :) 

With regards to folder import, it is the case that the import is "static" (i.e. it copies a given folder's content once). You do not need to remove an imported folder to _overwrite_ its content by going through the same import action. The re-import will overwrite content in the folder, so be aware of that to avoid field data entry losses.

**Reply by OBVYamaska** [reply]
Thank you very  much for pointing that out. I did look on QField's website before posting here and did not find this article... sorry!
Everything is clearer now. I'll inform my users accordingly!
